doc/api/globals.md

```diff
@@ -348,6 +359,17 @@ within the browser `var something` will define a new global variable. In
 Node.js this is different. The top-level scope is not the global scope;
 `var something` inside a Node.js module will be local to that module.
 
+## Class `Headers`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental. Enable this API with the [`--experimental-fetch`][]
+> CLI flag.
+
+A browser-compatible implementation of {Headers}.
```

Comment:
Any chance these could link to the relevant spec/mdn pages? (If they already do - my bad)

Reply:
(with an import rather than a global that is)

Reply:
they should, that's the changes to type-parser.mjs

Reply:
I think type-parser.mjs (cf. diff) takes care of that.